crmprtd/align.py

```diff
@@ -47,7 +47,7 @@ def closest_stns_within_threshold(sesh, network_name, lon, lat, threshold):
 
 
 def convert_unit(val, src_unit, dst_unit):
-    if src_unit != dst_unit:
+    if src_unit != dst_unit and src_unit is not None:
```

Comment:
What happens if `src_unit` *is* `None`? Or `dest_unit`? Seems to me that those should be error conditions (though I say so without looking at the calling context, so I'm happy to reconsider).